Prace_domowe/Praca_domowa7/Grupa1/Kacper_Grzymkowski/pd7.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<p>W pracy domowej chodziło raczej o coś takiego: https://www.kaggle.com/albertmistu/detect-anomalies-using-gmm. Niemniej jednak doceniam włożony trud - zadanie było nieoczywiste bo wymagało innego spojrzenia na problem niż to było na zajęciach.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>